versions.tf

```diff
@@ -1,3 +1,20 @@
 terraform {
-  required_version = "~> 1.8"
+  required_version = "~> 1.9"
```

Comment:
Hier sollte/müsste man beachtet, dass hier OpenTofu excludiert, da tofu immer eine Version hinterher hingt.

Abhilft könnte eine `versions_override.tofu` (https://opentofu.org/docs/language/files/override/) schaffen.

Reply:
We use _"enhances input variable validations"_ from Terraform 1.9 very heavily in this and future modules. As long as OpenTofu does not support this, we will not be able to support OpenTofu.